README.md

```diff
@@ -1,11 +1,60 @@
+[dco-badge-link]: https://github.com/opensearch-project/trace-analytics/actions/workflows/dco.yml
+[link-check-badge]:https://github.com/opensearch-project/trace-analytics/actions/workflows/link-checker.yml/badge.svg
+[link-check-link]: https://github.com/opensearch-project/trace-analytics/actions/workflows/link-checker.yml
+[dashboard-build-badge]: https://github.com/opensearch-project/trace-analytics/actions/workflows/dashboards-observability-test-and-build-workflow.yml/badge.svg
+[dashboard-build-link]: https://github.com/opensearch-project/trace-analytics/actions/workflows/dashboards-observability-test-and-build-workflow.yml
+[opensearch-build-badge]: https://github.com/opensearch-project/trace-analytics/actions/workflows/opensearch-observability-test-and-build-workflow.yml/badge.svg
+[opensearch-build-link]: https://github.com/opensearch-project/trace-analytics/actions/workflows/opensearch-observability-test-and-build-workflow.yml
+[dashboard-codecov-badge]: https://codecov.io/gh/opensearch-project/trace-analytics/branch/main/graphs/badge.svg?flag=dashboards-observability
+[opensearch-codecov-badge]: https://codecov.io/gh/opensearch-project/trace-analytics/branch/main/graphs/badge.svg?flag=opensearch-observability
+[codecov-link]: https://codecov.io/gh/opensearch-project/trace-analytics/branch/main/graphs/badge.svg?flag=dashboards-observability
+[cypress-test-badge]: https://img.shields.io/badge/Cypress%20tests-success-green
+[cypress-test-link]: https://github.com/opensearch-project/trace-analytics/blob/main/dashboards-observability/.cypress/CYPRESS_TESTS.md
+[opensearch-it-badge]: https://img.shields.io/badge/Plugin%20IT%20tests-success-green
+[opensearch-it-link]: https://github.com/opensearch-project/trace-analytics/blob/main/opensearch-observability/src/test/kotlin/org/opensearch/observability/ObservabilityPluginIT.kt
+[bwc-tests-badge]: https://img.shields.io/badge/BWC%20Tests-in%20progress-yellow
+[bwc-tests-badge]: https://github.com/opensearch-project/trace-analytics/issues/276
+[good-first-badge]: https://img.shields.io/github/issues/opensearch-project/trace-analytics/good%20first%20issue.svg
+[good-first-link]: https://github.com/opensearch-project/trace-analytics/issues?q=is%3Aopen+is%3Aissue+label%3A"good+first+issue"
+[bug-badge]: https://img.shields.io/github/issues/opensearch-project/trace-analytics/bug.svg 
+[bug-link]: https://github.com/opensearch-project/trace-analytics/issues?q=is%3Aopen+is%3Aissue+label%3Abug+
+[ehancement-badge]: https://img.shields.io/github/issues/opensearch-project/trace-analytics/enhancement.svg 
+[enhancement-link]: https://github.com/opensearch-project/trace-analytics/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement+
+
+
+### Event Analytics
+
+Event Analytics allows user to monitor, correlate, analyze and visualize machine generated data through Piped Processing Language. It also enables the user to turn data-driven events into visualizations and save frequently used ones for quick access. 
```

Comment:
maybe link the ppl doc here https://opensearch.org/docs/latest/search-plugins/ppl/index/